packages/bpk-component-autosuggest/src/BpkAutosuggest.js

```diff
@@ -49,10 +51,16 @@ Autosuggest.defaultProps.theme = {
   sectionTitle: getClassName('bpk-autosuggest__section-title'),
 };
 
-Autosuggest.defaultProps.renderInputComponent = inputProps => {
+type Props = {
```

Comment:
This one I had to take a rough guess based on other uses of ref (and the component which these props are used) in Backpack as I couldn't see the props in `Autosuggest` lib or in `BpkInput`